### ISSUE TITLE
customizable image whitelist

### ISSUE THREAD

**Issue by OneShot-Niko** [mybearworld/roarer#49]
I think i havent mentioned this yet so
Considering you have an option to allow any imagehost even outside the whitelist, I think it'd make sense if the user could whitelist CDNs or etc - However that is just me.

**Comment by mybearworld** [comment]
The problem I can imagine with this is that if everyone has a separate whitelist, there's no guarantee at all if an image will actually work

**Comment by OneShot-Niko** [comment]
> The problem I can imagine with this is that if everyone has a separate whitelist, there's no guarantee at all if an image will actually work

Yeah that could be a problem...

**Comment by mybearworld** [comment]
How would you use this? Maybe there's a better

**Comment by OneShot-Niko** [comment]
> How would you use this? Maybe there's a better way of doing this.

Yeah, now that I'd thought of it, this is probably not a really good path to do it either.
No idea how else though...
